x-pack/plugins/security_solution/server/endpoint/lib/artifacts/saved_object_mappings.ts

```diff
@@ -34,3 +34,31 @@ export const manifestType: SavedObjectsType = {
   mappings: manifestSavedObjectMappings,
   migrations,
 };
+
+export const unifiedManifestSavedObjectType = ManifestConstants.UNIFIED_SAVED_OBJECT_TYPE;
+
+export const unifiedManifestSavedObjectMappings: SavedObjectsType['mappings'] = {
+  dynamic: false,
+  properties: {
+    artifactIds: {
+      type: 'text',
+    },
+    policyId: {
+      type: 'keyword',
+    },
+    semanticVersion: {
+      type: 'text',
+    },
+    created: {
```

Comment:
should we use the created value from ES/SO document instead?

Reply:
Good question. I don't see why not but that's the way it's been implemented in the current manifest client. Does anyone who could have an opinion on this come to mind?

Reply:
I'd suggest to use the SO's own `created_at` field. It might be optional in the type in TS, but afaik it's always [filled](https://github.com/elastic/kibana/blob/main/packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/apis/create.ts#L134).

For uninstall tokens, we use SO's own `created_at`, that's for sure. (There was even an issue, when we had an own `created_at` field, because during querying the SOs, the two fields had a sort of a naming collision inside the SO implementation. : ) so eventually the own `created_at` were removed [here](https://github.com/elastic/kibana/pull/159985))

One drawback could be that we cannot modify the creation date through SO client - but I guess it's only needed during migration.

Reply:
Used created_at eda8443785d1900eea14540dbf7f0846c0315e36